test/distributed/tensor/parallel/test_tp_examples.py

```diff
@@ -239,7 +242,10 @@ def test_transformer_training(self, is_seq_parallel=False):
                 output_layouts=Shard(1)
             ) if is_seq_parallel else RowwiseParallel()
 
-            parallelize_module(layer, device_mesh, layer_parallelize_plan)
+
+            with CommDebugMode() as comm_mode:
+                parallelize_module(layer, device_mesh, layer_parallelize_plan)
+            self.assertDictEqual(comm_mode.get_comm_counts(), {})
```

Comment:
ditto: no need to track collectives on the parallelize_module parts

Reply:
done